src/components/Faq/FaqNav.js

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import { useLocation } from "@reach/router";
+
+import "./FaqNav.css";
+import Button from "../Button";
+
+const FaqNav = ({ navData, modifier }) => {
+  const modifierClass = modifier ? `FaqNav--${modifier}` : ``;
+  const location = useLocation();
+  const pathname = location.pathname;
+
+  return (
+    <div className={`FaqNav ${modifierClass}`}>
+      <nav>
+        <ul>
+          {navData.map((navItem, i) => {
+            const activeClass = pathname.includes(navItem.slug)
+              ? `is-active`
+              : ``;
+
+            return (
+              <li className={activeClass}>
+                <Button
+                  text={navItem.title_override}
+                  color="primary"
+                  href={`/refugee-guide${navItem.slug}`}
```

Comment:
Could this be a `to` prop instead of `href`? I would be better to keep the Gatsby pre-fetching functionality by using the internal `Link` if possible.

Reply:
Also, please use the trailing-slash version of the URL (`/path/` instead of `/path`).